playground.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
```

Comment:
please remove the playground file from the PR ;-)

Reply:
done